src/dev/i18n/extractors/handlebars.js

```diff
@@ -19,8 +19,8 @@
 
 import chalk from 'chalk';
 
-import { formatJSString } from './utils';
-import { createFailError } from '../run';
+import { formatJSString } from '../utils';
+import { createFailError } from '../../run';
```

Comment:
@azasypkin stacktrace is redundant for tool's exceptions, so I use `createFailError` for all of them to avoid ambiguous stacktrace. But I agree with the first comment.

Reply:
> @azasypkin stacktrace is redundant for tool's exceptions,

Just to make sure we're on the same page, I mean that [here](https://github.com/elastic/kibana/blob/master/src/dev/i18n/extract_default_translations.js#L136) we can just:
```js
throw createFailError(
   `${chalk.white.bgRed(' I18N ERROR ')} Error in ${normalizePath(name)}\n${error.message}`
);
```
where `error` is the one that is thrown from the extractors and can be just `Error` and we can drop stack trace here and include only message into rethrown error. Or I missing something?

Reply:
@azasypkin https://github.com/elastic/kibana/pull/22254#discussion_r213119205
Stacktrace of unexpected errors shouldn't be dropped.

Reply:
Aha, thanks! Then can you just throw `failErrors` from extractors without `I18n` prefix and use `combineErrors` (exported together with `createFailError`) in the top level catch then or it won't produce what we want to see?

Reply:
Oh, weird stuff, okay it's not as easy as I imagined. Please ignore this second part of my initial suggestion, sorry for the noise.